src/pygnssutils/gnssntripclient.py

```diff
@@ -311,20 +311,21 @@ def _app_get_coordinates(self) -> tuple:
         """
 
         lat = lon = alt = sep = 0.0
+        siv = rtk = age = 0
         if self._settings["ggamode"] == GGAFIXED:  # Fixed reference position
             lat = self._settings["reflat"]
             lon = self._settings["reflon"]
             alt = self._settings["refalt"]
             sep = self._settings["refsep"]
         elif self.__app is not None:
             if hasattr(self.__app, "get_coordinates"):  # live position from receiver
-                _, lat, lon, alt, sep = self.__app.get_coordinates()

```

Comment:
I appreciate the intention here, but unfortunately changing the footprint of the `get_coordinates()` function has knock-on effects in other external applications which are not addressed here, including [PyGPSClient](https://github.com/semuconsulting/PyGPSClient/blob/master/src/pygpsclient/app.py#L913).

In addition, I'm also not sure the additional GGA information would actually be used by any mainstream NTRIP Caster - to the best of my knowledge, NTRIP Casters which request GGA information are principally interested in the client lat/lon for geofencing purposes - the numSV and DGPS correction parameters would be redundant (_if you know otherwise, I'll happily stand corrected on this_).